rs/types/types/src/messages/inter_canister.rs

```diff
@@ -633,9 +641,9 @@ impl From<pb_queues::RequestMetadata> for RequestMetadata {
     fn from(metadata: pb_queues::RequestMetadata) -> Self {
         Self {
             call_tree_depth: metadata.call_tree_depth.unwrap_or(0),
-            call_tree_start_time: Time::from_nanos_since_unix_epoch(
-                metadata.call_tree_start_time_nanos.unwrap_or(0),
-            ),
+            call_tree_start_time: metadata
+                .call_tree_start_time_nanos
+                .map_or(UNIX_EPOCH, Time::from_nanos_since_unix_epoch),
```

Comment:
I'd add a TODO here (backed by a ticket) to replace this with `try_from_option_field()` in a future release. There should be no more `None` values in the checkpoint after this.

Reply:
I deprecated the optional fields and added non-optional ones. Added a todo to remove the optional fields with a [ticket](https://dfinity.atlassian.net/jira/software/c/projects/MR/boards/223?selectedIssue=MR-641).